ROADMAP.md

```diff
@@ -99,3 +99,28 @@ A plot showing annualized profit yields as a function of pool size.
 ### Follow-on extensions / analysis
 
 * Use the differing annualized profit yields as an input for an agent-based model to study the possible centralisation of validators in staking pools over time.
+
+## 5. Introduce live data to the model initial states
+
+### Description
+
+The current model uses static datasets for the initial states. An API is used to get the number of active validators and ETH staked balance, as the historical ETH supply and price come from a static CSV that needs to be updated regularly.
```

Comment:
`The current model uses static datasets for the initial states`

Suggest:
> The model currently has a number of State Variables and System Parameters that are initialized either using static values or calculated using static CSV files.

`An API is used to get the number of active validators and ETH staked balance, as the historical ETH supply and price come from a static CSV that needs to be updated regularly.`

> These static values need to be updated periodically to ensure the model properly represents the real-world system state, this is currently a manual process.

> For this model extension, we suggest refactoring a number of the State Variables and System Parameters to be updated from live data sources, and create a true digital twin.